toolbox/fdc3-conformance/FDC3-1.2-Conformance-Test-Cases.md

```diff
@@ -0,0 +1,190 @@
+-  `ACFilteredContext3`: Perform above test, except creating a _different_ channel. Check that you _don't_ receive anything.
+-  `ACUnsubscribe`: Perform above test, except that after creating the channel **A** then `unsubscribe()`s the listener it added to the channel. Check that **A** _doesn't_ receive anything.
+-  `ACFilteredContext4`: Perform above test, except that after creating the channel **A** creates another channel with a further _different_ channel id.  Check that **A** _doesn't_ receive anything.
+
+### App Channel History
+
+| App | Step               | Details                                                                                                                                                              |
+|-----|--------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| A   | 1. createChannel        |`fdc3.getOrCreateChannel("test-channel")`       |
+| B   | 2. createChannel        | `fdc3.getOrCreateChannel("test-channel")`   |
+| B   | 3. Broadcast          | `testChannel.broadcast()` the instrument context.<br>`testChannel.broadcast()` a contact context. |
+| A   | 4. Receive Context    | `testChannel.getCurrentContext('fdc3.instrument')` returns the last broadcast instrument<br>`testChannel.getCurrentContext('fdc3.contact')` returns the last broadcast contact                                                              |
+
+-  `ACContextHistoryTyped`: Perform above test.
+-  `ACContextHistoryMultiple`: **B** Broadcasts multiple history items of both types.  Only the last version of each type is received by **A**.
+-  `ACContextHistoryLast`: **A** calls `testChannel.getCurrentContext()` retrieves the last broadcast context item
+
+## 4. Open API 
+
+### A Opens B
+
+- `AOpensB1`:  **A** calls `fdc3.open(‘app B Name’)`, check app **B** opens
+- `AOpensB2`:  **A** calls `fdc3.open({name: “<app B Name>”})`, check app **B** opens
+- `AOpensB3`:  **A** calls `fdc3.open({name: “<app B Name>”, appId: “<app B ID”})`, check app **B** opens
+
+### A Fails To Open B
+
+- `AFailsToOpenB1-3`:  Run the above 4 tests again with a non-existent app name/app id.  Should return “App Not Found” Error from https://fdc3.finos.org/docs/api/ref/Errors#openerror
+
+### A Opens B With Context
+
+| App | Step            | Description                                              |
+|-----|-----------------|----------------------------------------------------------|
+| A   | 1. Opening App     | various open methods as in `AOpensB1-3` except with a `<context>` argument of type `fdc3.testReceiver` <br>check app opens    |
+| B   | 2. Context present | `fdc3.addContextListener()`<br>- receives `<context>` from **A** |
+
+-  `AOpensBWithContext1-3`: Perform above tests
+-  `AOpensBWithSpecificContext`: Perform above but replace **B**s call with `fdc3.addContextListener('fdc3.testReceiver`)`
```

Comment:
The other tests in this group have 3 variations. This doesn't appear to (and probably doesn't need to) - so which variation should it be using (doesn't matter, but be specific)?